src/kudu/consensus/consensus_queue.cc

```diff
@@ -1212,10 +1223,7 @@ int64_t PeerMessageQueue::ComputeNewWatermarkDynamicMode(int64_t* watermark) {
   CHECK(queue_state_.active_config->commit_rule().mode() ==
       QuorumMode::SINGLE_REGION_DYNAMIC);
 
-  // Double check that the local leader has the proper metadata.
-  CHECK(local_peer_pb_.has_attrs() && local_peer_pb_.attrs().has_region());
```

Comment:
i think this CHECK is still valid? We want region property to be always present?